.github/workflows/main.yml

```diff
@@ -745,7 +785,6 @@ jobs:
           name: ${{runner.os}}-${{env.COVERAGE_REPORTS}}
           path: .nyc_output
           retention-days: 1
-          if: always()
```

Comment:
if is not valid on the with statement, it's already on this job. This is creating warnings in the test runs for invalid syntax.